docs/Analyzer Configuration.md

```diff
@@ -654,17 +656,17 @@ Configurable Rules: [CA1010](https://docs.microsoft.com/dotnet/fundamentals/code
 Option Values: List (separated by '|') of interface names with their required generic fully qualified interface (separated by '->').
 Allowed interface formats:
 
-  1. Interface name only (includes all interfaces with the name, regardless of the containing type or namespace)
-  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format) with an optional "T:" prefix.
+  1. Interface name only (includes all interfaces with the name, regardless of the containing type or namespace).
+  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format) with an optional `T:` prefix.
 
-Default Value: None
+Default Value: _None_
 
 Examples:
 
 | Option Value | Summary |
 | --- | --- |
-|`dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`\``1` | All types implementing 'ISomething' regardless of its namespace are expected to also implement 'System.Collections.Generic.IEnumerable`1'. |
-|`dotnet_code_quality.CA1010.additional_required_generic_interfaces = T:System.Collections.IDictionary->T:System.Collections.Generic.IDictionary`\``2` | All types implementing 'System.Collections.Generic.IDictionary' are expected to also implement 'System.Collections.Generic.IDictionary`2'. |
+| ``dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`1`` | All types implementing 'ISomething' regardless of its namespace are expected to also implement 'System.Collections.Generic.IEnumerable\`1'. |
```

Comment:
```suggestion
| ``dotnet_code_quality.CA1010.additional_required_generic_interfaces = ISomething->System.Collections.Generic.IEnumerable`1`` | All types implementing 'ISomething' regardless of its namespace are expected to also implement ``System.Collections.Generic.IEnumerable\`1``. |
```

Reply:
Another example where it is shown correctly in GitHub but not in VS Code.